src/sfizz/SfzHelpers.h

```diff
@@ -240,7 +241,7 @@ constexpr void addToBase(T& base, T modifier)
  * @param base
  * @param modifier
  */
-constexpr void multiplyByCents(float& base, int modifier)
+void multiplyByCents(float& base, int modifier)
```

Comment:
this should be inline, otherwise this function will be defined&used more than once